runelite-client/src/main/java/net/runelite/client/plugins/hydra/HydraPrayerAttackCounterOverlay.java

```diff
@@ -26,43 +27,55 @@
 
 import java.awt.Dimension;
 import java.awt.Graphics2D;
+import java.util.Objects;
 import javax.inject.Inject;
 import javax.inject.Singleton;
+import net.runelite.api.NPC;
 import net.runelite.client.ui.overlay.Overlay;
 import net.runelite.client.ui.overlay.OverlayPosition;
 import net.runelite.client.ui.overlay.OverlayPriority;
 import net.runelite.client.ui.overlay.components.LineComponent;
 import net.runelite.client.ui.overlay.components.PanelComponent;
 
 @Singleton
-public class BabyHydraIndicatorOverlay extends Overlay
+public class HydraPrayerAttackCounterOverlay extends Overlay
 {
-	private final BabyHydraPlugin plugin;
+	private final HydraPlugin hydraPlugin;
 
-	private final PanelComponent panelComponent = new PanelComponent();
+	private final PanelComponent panelComponent;
 
 	@Inject
-	private BabyHydraIndicatorOverlay(final BabyHydraPlugin plugin)
+	private HydraPrayerAttackCounterOverlay(HydraPlugin hydraPlugin)
 	{
-		this.plugin = plugin;
+		this.hydraPlugin = hydraPlugin;
+		this.panelComponent = new PanelComponent();
+		this.panelComponent.setPreferredSize(new Dimension(14, 0));
 		setPosition(OverlayPosition.BOTTOM_RIGHT);
 		setPriority(OverlayPriority.MED);
-		panelComponent.setPreferredSize(new Dimension(14, 0));
 	}
 
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
-		if (plugin.getHydra() != null && plugin.getHydras().containsKey(plugin.getHydra().getIndex()))
+		NPC npc = hydraPlugin.getInteractingNpc();
+
+		if (Objects.isNull(npc))
```

Comment:
Any reason for `Objects.isNull` over just `object == null`?

Reply:
Nope. I've changed them. Seems Objects methods is really only intended for lambda expressions.